src/lib.rs

```diff
@@ -128,6 +132,20 @@ pub fn collect_stars(config: Config) -> Result<(), Box<dyn error::Error>> {
     }
     println!("Collected {} stars", stars.len());
 
+    let path = Path::new("output.md");
+    let display = path.display();
+
+    let mut file = match File::create(&path) {
```

Comment:
Since `path` isn't used anywhere else, we can inline its value; `File::create` can accept as an argument [anything that implements the `AsRef<Path>` trait](https://doc.rust-lang.org/std/fs/struct.File.html#method.create) which [`str` does](https://doc.rust-lang.org/std/primitive.str.html#impl-AsRef%3Cstr%3E).

See the `File::create` example [in the docs](https://doc.rust-lang.org/std/fs/struct.File.html#examples-2).